lib/livebook/file_system/s3.ex

```diff
@@ -378,15 +412,15 @@ defimpl Livebook.FileSystem, for: Livebook.FileSystem.S3 do
       region: fields["region"],
       access_key_id: fields["access_key_id"],
       secret_access_key: fields["secret_access_key"],
-      prefix: fields["prefix"]
+      hub_id: fields["hub_id"]
```

Comment:
It's fine to use the virtual field to compute the initial id, but we should store and load the id rather than `hub_id` :)

Reply:
Should we compute the `hub_id` for Personal hub? I guess it would make sense like we did with Secrets. And it would help for `startup_file_systems`.

Reply:
Aligning with secrets sounds reasonable, yeah.